addons/mail/models/discuss/discuss_channel.py

```diff
@@ -777,6 +769,26 @@ def _find_or_create_persona_for_channel(self, guest_name, timezone, country_code
             self.add_members(guest_ids=guest.ids, post_joined_message=post_joined_message)
         return self.env.user.partner_id if not guest else self.env["res.partner"], guest
 
+    def _channel_basic_info(self):
+        self.ensure_one()
+        return {
+            'avatarCacheKey': self._get_avatar_cache_key(),
+            'channel_type': self.channel_type,
+            'memberCount': self.member_count,
+            'id': self.id,
+            'name': self.name,
+            'defaultDisplayMode': self.default_display_mode,
+            'description': self.description,
+            'uuid': self.uuid,
+            'state': 'open',
```

Comment:
this makes no sense here either

state and is_minimized default values should stay inside channel_info